sbin/start-slave.sh

```diff
@@ -17,8 +17,14 @@
 # limitations under the License.
 #
 
-# Usage: start-slave.sh <worker#> <master-spark-URL>
-#   where <master-spark-URL> is like "spark://localhost:7077"
+# Starts a slave on the machine this script is executed on.
+
+usage="Usage: start-slave.sh <worker#> <spark-master-URL> where <spark-master-URL> is like "spark://localhost:7077"
+
+if [ $# -le 1 ]; then
```

Comment:
Yeah, "-lt 2" will be better.
I found `spark-daemon.sh`/`spark-daemons.sh` and other scripts also just use `usage=...`, so I think it is fine to keep it just like this.